source/templates.tex

```diff
@@ -2103,7 +2103,7 @@
 \grammarterm{requires-clause}{s},
 or
 \grammarterm{noexcept-specifier}{s}.
-For the purpose of instantiation, the substatements of a constexpr if
+For the purpose of instantiation, the substatements of a constexpr \tcode{if}
```

Comment:
This change is incorrect; we define the (non-teletype) term "constexpr if statement" in [stmt.if]p2.

Reply:
Reverted (changes to templates.tex are gone; they all dealt with constexpr if statement).

(Similar to "contextually converted to bool", it might be worthwhile to consider changing "constexpr if statement" to teletype, but italics teletype is ugly, and we never finished discussing that aspect. So ignore for now.)